src/tests/ftest/osa/osa_offline_extend.py

```diff
@@ -22,83 +24,190 @@ def setUp(self):
         """Set up for test case."""
         super().setUp()
         self.dmg_command = self.get_dmg_command()
+        self.daos_command = DaosCommand(self.bin)
         # Start an additional server.
+        self.ior_test_sequence = self.params.get("ior_test_sequence",
+                                                 '/run/ior/iorflags/*')
         self.extra_servers = self.params.get("test_servers",
                                              "/run/extra_servers/*")
+        self.rank = self.params.get("rank_list", '/run/test_ranks/*')
+        self.test_oclass = None
+        self.dmg_command.exit_status_exception = True
 
-    def run_offline_extend_test(self, num_pool, data=False):
+    def run_offline_extend_test(self, num_pool, data=False,
+                                server_boot=False, oclass=None):
         """Run the offline extend without data.
 
         Args:
             num_pool (int) : total pools to create for testing purposes.
             data (bool) : whether pool has no data or to create
                           some data in pool. Defaults to False.
+            server_boot (bool) : Perform system stop/start on a rank.
+                                 Defaults to False.
+            oclass (list) : list of daos object class (eg: "RP_2G8")
         """
         # Create a pool
         pool = {}
-        pool_uuid = []
+        tmp = server_boot
```

Comment:
Not sure why you need to use this "tmp" variable. "server_boot" is more intuitive.

Reply:
hmm... Probably, I started thinking about something and left it... Yes, there is no need for tmp. Will fix it. Thanks.